scss/mixins/_buttons.scss

```diff
@@ -73,7 +73,6 @@
       min-height: inherit;
       content: "";
       background-color: currentcolor;
-      -webkit-mask: #{$icon} no-repeat #{$position} / #{$size}; // stylelint-disable-line property-no-vendor-prefix
```

Comment:
Sure about this one? Asking because it wasn't mentioned in the issue nor in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1574/files. Just to double-check there's no regression.

Reply:
Found it here: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1574#discussion_r1006434287

Reply:
OK just wanted to be sure that `$icon` contained also a problematic `--o` custom property in this case. Resolving this comment thx.